app/models/jam.rb

```diff
@@ -33,6 +35,12 @@ def duration
 
   private
 
+  def make_midi_solo
+    if ['audio/midi', 'audio/x-midi'].include?(file.content_type)
+      self.jam_type_list = "Solo"
```

Comment:
we should stick to either `'` or `"` (did I say `rubocop` before? 😈) since you are mixing both styles in this method